src/compatibility.h

```diff
@@ -83,6 +83,25 @@
  * - Removes OID column
  */
 #if PG_VERSION_NUM >= 120000
+
```

Comment:
Probably not. I thought about changing that but decided not to, mostly because of how it was structured before, i.e. 9.4 had its own #ifdef block even though it was the min version.

I'm not really that satisfied about how this is organized. In particular I don't like that you need to add #ifndef to prior sections when adding a new version. I'd prefer if each macro got its own #ifdef block,  e.g. https://github.com/pgbackrest/pgbackrest/blob/release/2.53/src/postgres/interface/version.vendor.h#L102. But that's a larger change than I want to make at this time.

Having said all that, I'll pull out the `#if PG_VERSION_NUM >= 120000` block if you think that's the way to go.

Reply:
Not a big deal to me, was just curious why it was still there. If it's easier to take care of as part of a larger refactor later, I'm fine with that